sage-ios/SAGE/SAGE/EditAnnouncementController.swift

```diff
@@ -52,4 +58,22 @@ class EditAnnouncementController: AddAnnouncementController {
         }
     }
 
+    func deleteAnnouncement(sender: UIButton!) {
+        let view = (self.view as! AddAnnouncementView)
+        view.button.startLoading()
```

Comment:
can you also call startLoading and stopLoading on the finishButton wherever you do this for the deleteButton so that the user cant make a request to edit the view controller? we wanna limit what they can do while a network request is being made